docs/integrations/persisting-store-data.md

```diff
@@ -291,12 +291,12 @@ export const useBoundStore = create(
 ```
 
 ```tsx
-export function StoreConsumer(){
-  const store = useBoundStore();
+import { useBoundStore } from './path-to-store';
 
+export function StoreConsumer(){
   // hydrate persisted store after on mount
   useEffect(() => {
-    store.persist.reHydrate();
+    useBoundStore.persist.reHydrate();
```

Comment:
TS: "Property 'reHydrate' does not exist on type (...). Did you mean 'rehydrate'?"

```suggestion
    useBoundStore.persist.rehydrate();
```

Reply:
Please also fix this typo in line 276.
"and it is left up to you to manually call `reHydrate()`."

Reply:
resloved